module/zcommon/zfeature_common.c

```diff
@@ -217,8 +217,11 @@ zfs_mod_supported_feature(const char *name)
 	 * libzpool, always supports all the features. libzfs needs to
 	 * query the running module, via sysfs, to determine which
 	 * features are supported.
+	 *
+	 * The equivalent _can_ be done on FreeBSD by way of the sysctl
+	 * tree, but this has not been done yet.
```

Comment:
It might be worth while keeping this in mind to add it as an issue to the issue tracker after the freebsd merger.

Reply:
I've added a note to the OpenZFS project for this item.